code/modules/atmospherics/machinery/components/unary_devices/outlet_injector.dm

```diff
@@ -61,14 +61,18 @@
 	if(!on || !is_operational)
 		return
 
+	var/turf/location = get_turf(loc)
```

Comment:
Shouldn't the loc always be a turf?

Reply:
bump

Reply:
I forgot to attach here that we talked about this on discord 
![Screenshot_20210404-081922_Discord](https://user-images.githubusercontent.com/42839747/113500481-8e117b80-951e-11eb-811f-902d92e8d079.jpg)

Reply:
I hate old me